data/core/editor/items.cfg

```diff
@@ -216,6 +325,12 @@
         name= _ "Flaming Sword"
     [/item]
 
+    [item]
+        id=flame-sword-bare
+        image=items/flame-sword-bare.png
+        name= _ "Bare Flaming Sword"
```

Comment:
It looks a lot more like an ice-sword than anything to do with flames.

Reply:
If we have to go add fancy words, I suggest "Unenchanted"/"Disenchanted" Flame Sword.